Fix errors when building the info manual:

[...]
./screen.texinfo:5799: unknown command `suse'
./screen.texinfo:5800: unknown command `deuxchevaux'
make[2]: *** [Makefile:31: screen.info] Error 1
[...]

Patch copied from upstream source repository:

http://git.savannah.gnu.org/cgit/screen.git/commit/?id=a7d2f9e6ecfa794dd0bd3dbeaf6780c88a6f3152

From a7d2f9e6ecfa794dd0bd3dbeaf6780c88a6f3152 Mon Sep 17 00:00:00 2001
From: Andreas Stieger <astieger@suse.com>
Date: Tue, 28 Feb 2017 19:23:01 +0100
Subject: [PATCH] fix texinfo syntax errors

thanks to Andreas Stieger <astieger@suse.com>
---
 src/doc/screen.texinfo | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/doc/screen.texinfo b/src/doc/screen.texinfo
index bc4cbae..aadad33 100644
--- a/doc/screen.texinfo
+++ b/doc/screen.texinfo
@@ -5795,7 +5795,7 @@ and Alexander Naumov <alexander_naumov@@opensuse.org>.
 Contributors @*
 ============
 
-@example
+@verbatim
      Thomas Renninger <treen@suse.com>,
      Axel Beckert <abe@deuxchevaux.org>,
      Ken Beal <kbeal@@amber.ssd.csd.harris.com>,
@@ -5826,7 +5826,7 @@ Contributors @*
      Jason Merrill <jason@@jarthur.Claremont.EDU>,
      Johannes Zellner <johannes@@zellner.org>,
      Pablo Averbuj <pablo@@averbuj.com>.
-@end example
+@end verbatim
 
 @noindent
 Version @*
-- 
cgit v1.0-41-gc330

